NMS-14455: Install JDBC Driver on Core (#5113)
OpenNMS/opennms: docs/modules/deployment/pages/core/getting-started.adoc

endif::[]
====

== Receive SNMP traps/informs

> OpenNMS {page-component-title} Core lets you receive and process SNMP traps/informs out of the box.
> The OpenNMS {page-component-title} Core services run as an unprivileged user and can't bind on port numbers below 1024 without escalated privileges.

Reviewer: Is "Core" an official term we are using as a proper noun, or a term used in the docs to differentiate from Minion/Sentinel.?
```suggestion
{page-component-title} Core lets you receive and process SNMP traps/informs out of the box.
{page-component-title} services run as an unprivileged user and can't bind on port numbers below 1024 without escalated privileges.
```

Reviewer 2: That's a good point. There is no official "Core" term, it's just to differentiate between Minion/Sentinel and the core ... it should be lower case.

Author: Good to know - I'll go through and correct it!

endif::[]
====

> You can verify your firewall and port forwarding configuration by sending an SNMP trap from a remote system to your OpenNMS {page-component-title} Core instance:

Reviewer: ```suggestion
You can verify your firewall and port forwarding configuration by sending an SNMP trap from a remote system to your {page-component-title} Core instance:
```

[source, console]
----
snmptrap -v 2c -c public opennms-core-host '' 1.3.6.1.4.1.2021.991.17 .1.3.6.1.2.1.1.6.0 s "Milky Way"<1><2>
----
<1> By default, OpenNMS uses the community string `public`.

> If you changed the community string in OpenNMS, use that name here.

Reviewer: ```suggestion
If you changed the community string in {page-component-title}, use that name here.
```

> <2> Replace `opennms-core-host` with the IP or FQDN of your OpenNMS {page-component-title} Core instance.

Reviewer: ```suggestion
<2> Replace `opennms-core-host` with the IP or FQDN of your {page-component-title} Core instance.
```

On RHEL and CentOS, the `snmptrap` command line tool is part of the `net-snmp-utils`.

> If you run on Debian or Ubuntu, you have to install the `snmp-utils` package.

Reviewer: ```suggestion
If you run on Debian or Ubuntu, the tool is part of the `snmp-utils` package.
```

endif::[]
====

You can verify your firewall and port forwarding configuration by sending an SNMP trap from a remote system to your {page-component-title} core instance:

[source, console]
----
snmptrap -v 2c -c public opennms-core-host '' 1.3.6.1.4.1.2021.991.17 .1.3.6.1.2.1.1.6.0 s "Milky Way"<1><2>
----

> <1> By default, OpenNMS uses the community string `public`.

Reviewer: ```suggestion
<1> By default, {page-component-title} uses the community string `public`.
```
